sonar-go-to-slang/go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/SonarSource/slang/sonar-go-to-slang
 
-go 1.21
+go 1.23.4
```

Comment:
As new `range` options are syntactically the same and we don't have any advanced analysis in the Go analyzer, no changes in the parser and/or converter are needed